tools/tertiary-analysis/decoupler/decoupler_pathway_inference.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="decoupler_pathway_inference" name="Decoupler Pathway Inference" version="1.4.0+galaxy0" profile="20.05" license="MIT">
+    <description>
+        Pathway inference using the Decoupler.
+    </description>
+    <requirements>
+        <requirement type="package" version="1.4.0">decoupler</requirement>
+    </requirements>
+    <command>
+        python '$__tool_directory__/decoupler_pathway_inference.py'
+            -i '$input_anndata'
+            -n '$input_network_file'
+            --min_n "$min_n"
+            $use_raw
+            --output "inference"
+            $write_activities_path
+    </command>
+    <inputs>
+        <param name="input_anndata" type="data" format="h5ad" label="Input AnnData file" />
+        <param name="input_network_file" type="data" format="tabular" label="Input Network file" />
```

Comment:
Pleas add a `help="...."` briefly explaining the format of this file (for instance, first column includes geneset/pathwat name, second column should include gene identifiers matching the var index used in the AnnData - not sure if this text is the correct, but you get the idea). You could also add a formatted example in the `help` section after tests.

Reply:
This is now fixed, added in help(s)